crates/shared/src/price_estimation/factory.rs

```diff
@@ -203,6 +201,7 @@ impl<'a> PriceEstimatorFactory<'a> {
             optimal,
             fast,
```

Comment:
I think this means we no longer need a distinction between `optimal` and `fast`, but rather `verified` and `unverified` right? I would change the entry to only have those two (instead of having the three `optimal`, `fast` and `verified`).

Also, I refactored the `PriceEstimatorCreating` trait so that it can be used for co-located driver estimators, and so we don't have separate code paths for traditional and driver-based estimator creation. See #1521

Reply:
> I think this means we no longer need a distinction between optimal and fast, but rather verified and unverified right?

I think this is not true (at least for now).
1. Verifying all quotes that previously were `optimal` is too much for prod. So we still need a way to get the cheaper to compute `optimal` quotes.
2. Not all estimators that currently have a verified implementation so `verified` would contain fewer estimators than `optimal`. This means `verified` is (as of now) **not** just a stricter version of `optimal`; it's slightly different.

Reply:
> This means verified is (as of now) not just a stricter version of optimal; it's slightly different.

This I agree completely, my point is that now `optimal` and `fast` estimators are the same:

```rust
        let optimal = fast.clone();
```

So, instead of having `optimal` and `fast` estimators for each entry, we can have:

```rust
#[derive(Clone)]
struct EstimatorEntry {
    unverified: Arc<dyn PriceEstimating>,
    verified: Option<Arc<dyn PriceEstimating>>,
    native: Arc<dyn PriceEstimating>,
}
```

And just construct `fast` and `optimal` `RacingPriceEstimator`s differently.